### ISSUE TITLE
Athom-cli not providing usefull debug infomation "node DeprecationWarning"

### ISSUE THREAD

**Issue by gruijter** [athombv/homey-issues-dp#80]
**What is your Homey version (Settings → System)?**

1.5.0 alpha 5

**What did you try to achieve?**

debug a new app

*Please include detailed steps how we can reproduce the issue*

sideload the app via athom project --run

**What was the result?**
```
1/4 - Archiving...
2/4 - Uploading to Homey @ 10.0.0.99:80...
3/4 - Running `com.gruijter.netgear`, press CTRL+C to abort...
(node:7804) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
4/4 - Debugging...
```

**What did you expect as result?**

That the warning provides usefull information to find out where/which function call is causing the issue

**Any other remarks**

Even after going through all my code in detail I still do not know where the cause of this warning lies...

**Comment by BasKiers** [comment]
Hi,

Although this is not a breaking bug it is annoying. I've created a pr that fixes this in athom-cli here: https://github.com/athombv/node-athom-cli/pull/34

Regards,
Bas

**Comment by gruijter** [comment]
confirmed fixed in athom-cli 1.0.41. Thx 👍
